experimental/LieAlgebras/src/simple_lie_algebra.jl

```diff
@@ -0,0 +1,272 @@
+@attributes mutable struct SimpleLieAlgebra{C<:RingElement} <: LieAlgebra{C}
+#Construct a simple Lie algebra over a given ring with a given root system
+  base_ring::Ring
```

Comment:
```suggestion
  R::Ring
```
you call it `R` elsewhere, and this would then be analogous to the other Lie algebra types in Oscar.